### DISCUSSION TITLE
RunID is coming as manual even if the DAG is being scheduled using TriggerDagRunOperator

### DISCUSSION THREAD

**Discussion by ashish1471** [apache/airflow#29231] [General]
I am using Airflow 2+. I am triggering my DAG automatically using schedule time and TriggerDagRunOperator. My run_id is coming as manual_(time) but I am expecting it to come as scheduled or something different from manual. so that, I can distinguish it with manual. Any leads would be appreciated?

**Comment by potiuk** [comment]
TrigerDagRun is a manual way of triggering. Schedule is when the run is triggered on well ... schedule. 

In recent versions of Airflow (can't remember but you can look for it in the changelog) you can specify your own run id when triggering Dag run via TriggerDagRunOperator - and this is the way how you can distinguish those runs (by using your own convention).
